static/js/ui_init.js

```diff
@@ -150,6 +150,34 @@ function initialize_right_sidebar() {
     });
 
     $("#right-sidebar-container").html(rendered_sidebar);
+
+    $("#user_presences").on("mouseover", ".user_sidebar_entry", (e) => {
+        const status_emoji = $(e.target).closest(".user_sidebar_entry").find("img.status_emoji");
+        if (status_emoji.length) {
+            const img_src = status_emoji.attr("src");
+            const emoji_name = status_emoji.data("emoji-name");
+            if (!img_src.endsWith(".gif") && emoji_name) {
+                const emoji_info = emoji.get_emoji_details_by_name(emoji_name);
+                if (emoji_info.url) {
+                    status_emoji.attr("src", emoji_info.url);
+                }
+            }
+        }
+    });
```

Comment:
I think these functions could be simpler if we just rendered into the template some `data-animated-url` and `data-still-url` attributes that can be used to update the display without doing a lookup.